.env.schema

```
@@ -52,6 +52,14 @@ MEMBERSHIP_API_TIMEOUT=30
 # @type=number @optional
 MEMBERSHIP_API_CHECK_INTERVAL=15
 
+# Email transport (django-environ email URL). Schemes: consolemail:// (default),
+# smtp://… for real delivery, filemail:///path to capture each mail to a file.
+# Plain string, not @type=url — the custom schemes don't parse as URLs.
+# @optional
+EMAIL_URL=consolemail://
+# @optional
+DEFAULT_FROM_EMAIL=Zagrajmy <noreply@zagrajmy.net>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEFAULT_FROM_EMAIL key should go before the EMAIL_URL key

(UnorderedKey)